src/philips_hue.py

```diff
@@ -131,5 +131,5 @@ def trigger_light_from_monitoring_notification(client, notification, light_id):
         closed_state = PhilipsHueState.CLOSED
```

Comment:
What if you put the philips files in a subdirectory called something like: `external_integrations`, `third_party_integrations`, `client`, etc.

This is to prepare for the addition of the jira code.